verifier/src/test/kotlin/itmo/verifier/FormulaParserTest.kt

```diff
@@ -0,0 +1,81 @@
+package itmo.verifier
+
+import AU
+import CTLGrammar
+import EU
+import EX
+import Element
+import Not
+import Or
+import TRUE
+import com.github.h0tk3y.betterParse.grammar.parseToEnd
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class FormulaParserTest {
+
+    @Test
+    fun `true test`() {
+        val formulaString = "1"
+
+        assertEquals(CTLGrammar.parseToEnd(formulaString), TRUE)
+    }
+
+    @Test
+    fun `and test`() {
+        val formulaString = "1 & 0"
+
+        assertEquals(CTLGrammar.parseToEnd(formulaString), Not(Or(Not(TRUE), Not(Not(TRUE)))))
```

Comment:
Not sure, but probably things like `Not(Not(...))` will have a bad effect on performance at complex formula

Reply:
Yes, I also thought about this
Decided not to put this in grammar, it seems not really it's matter 
But I'll write the optimization function inside the CTLFormula class